lib/livebook/live_markdown/import.ex

```diff
@@ -354,7 +354,8 @@ defmodule Livebook.LiveMarkdown.Import do
       if is_nil(hub_id) or Hubs.hub_exists?(hub_id) do
         {attrs, true, messages}
       else
-        {Map.delete(attrs, :hub_id), false, messages ++ [@unknown_hub_message]}
+        {Map.drop(attrs, [:hub_id, :deployment_group_id]), false,
```

Comment:
What if the hub is valid, but the deployment group no longer exists?

Reply:
No problem in this case

Reply:
We are checking inside the component, so it will fallback to `none`